tools/ports/libpng.py

```diff
@@ -7,8 +7,9 @@
 import shutil
 import logging
 
+# TODO - update these once upstream is current
 TAG = 'version_1'
```

Comment:
This seems odd that the tag stayed the same but the HASH of the content didn't change?   I would expect this to change to `version_2`, or something other than `version_1`

Reply:
Fixed indirectly with the Google storage fix.